.gitignore

```diff
@@ -1,3 +1,4 @@
-.cog/
+.vscode/
```

Comment:
protip: add this to your global git config so you don't have to add it to all repos. https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
```suggestion
```